lib/mini_mime.rb

```diff
@@ -44,7 +49,7 @@ def self.lookup_by_filename(filename)
         extension.downcase!
         if extension.length > 0
           LOCK.synchronize do
-            @db ||= new
+            @db ||= defined?(MicroMime) ? MicroMime : new
```

Comment:
Note that ultimately we'll want this backend to be configurable, even if only so we can run our test suite with/without MicroMime enabled.